routes/ci.js

```diff
@@ -0,0 +1,126 @@
+var express = require("express");
+var router = express.Router();
+var request = require("request")
+var repositoryModel = require("../model/repository.js");
+var crypter = require("../util/crypter.js");
+var generator = require("../backend/generator.js");
+var deploy = require("../backend/deploy.js");
+router.get('/register', function (req, res, next) {
+  request({
+    url: `https://api.github.com/users/${req.session.username}/repos?per_page=100`,
+    headers: {
+      'User-Agent': 'request'
+    }
+  },
+  function (error, response, body) {
+    var repos = JSON.parse(body);
+    res.render("index", {
```

Comment:
do res.redirect("/?success=true") and do something similar to what i did at ownCloud
https://github.com/owncloud/core/pull/27277/files#diff-5af29c9f0fdb4338457f3bb7024dc991R152